src/libcrun/handlers/krun.c

```diff
@@ -50,16 +50,34 @@ struct krun_config
   void *handle;
   void *handle_sev;
   bool sev;
+  int32_t ctx_id;
+  int32_t ctx_id_sev;
 };
 
 /* libkrun handler.  */
 #if HAVE_DLOPEN && HAVE_LIBKRUN
+static int32_t
+libkrun_create_context (void *handle, libcrun_error_t *err)
+{
+  int32_t (*krun_create_ctx) ();
+  int32_t ctx_id;
+
+  krun_create_ctx = dlsym (handle, "krun_create_ctx");
+  if (krun_create_ctx == NULL)
+    return crun_make_error (err, 0, "could not find symbol in `libkrun.so`");
```

Comment:
```libkrun.so``` might sometimes be ```libkrun-sev.so``` in the error message

Reply:
Would it be reasonable to remove the `.so` to avoid inferring we're talking about that specific library, or do we need it to point to the right library?

Reply:
I think we can just change the error message `in the krun library`, or something like that